pyproject.toml

```diff
@@ -30,8 +30,13 @@ httpx = "^0.18.2"
 hypothesis = "^6.14.0"
 black = "^21.6b0"
 watchgod = "^0.7"
-
+taskipy = "^1.9.0"
 
 [build-system]
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
+
+[tool.taskipy.tasks]
+start = "uvicorn api.main:app --host 0.0.0.0 --port 8000 --reload"
+lint = "black . --check --diff && flake8 . --diff"
```

Comment:
I would suggest moving them to a separate task, something like --

```toml
[tool.taskipy.tasks]
black = { cmd = "black --check .", help = "dry run of black" }
flake8 = { cmd = "flake8 .", help = "Lints code with flake8" }
```

Reply:
I noticed we are missing a `pre-commit` setup for the project, was it just not looked at or there isn't a need as of now?